public/ifc2sql.py

```
@@ -0,0 +1,707 @@
+                database.parent.mkdir(parents=True, exist_ok=True)
+            else:
+                # Assume it's a filepath - existing or not.
+                pass
+
+            if database.suffix.lower() not in (".sqlite", ".db", ".ifcsqlite"):
+                database = database.with_suffix(database.suffix + ".sqlite")
+            database = str(database)
+        elif self.sql_type == "mysql":
+            database = self.database
+        else:
+            assert False
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)